base/gfspconfig/config.go

```diff
@@ -169,6 +169,9 @@ type ParallelConfig struct {
 	ChallengePieceParallelPerNode       int
 	AskReplicateApprovalParallelPerNode int
 	QuerySPParallelPerNode              int64
+
+	GlobalDiscontinueTimeInterval  int
```

Comment:
may be add DisenableDiscontinueBucket (bool) config?
in test net it is false, will start DiscontinueBucket.
in main net it can be set true.

Reply:
yes, I can add the config